LSP server fails to initialise in Helix editor

When using `standardrb --lsp` in Helix I get error messages in the log:

```
2023-03-31T18:05:48.225 helix_lsp [ERROR] failed to initialize language server: failed to parse: invalid type: boolean `true`, expected struct ExecuteCommandOptions
```

As this worked in Neovim, I initially reported the bug to Helix: https://github.com/helix-editor/helix/issues/6506 who responded:

> This is a bug in StandardRB: it's sending "executeCommandProvider":true in the initialization response but that field can't be a boolean in the spec: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#executeCommandOptions

--- a/lib/standard/lsp/routes.rb
+++ b/lib/standard/lsp/routes.rb
@@ -28,7 +28,9 @@ module Standard
           capabilities: Proto::Interface::ServerCapabilities.new(
             document_formatting_provider: true,
             diagnostic_provider: true,
-            execute_command_provider: true,
+            execute_command_provider: Proto::Interface::ExecuteCommandOptions.new(
+              commands: ["standardRuby.formatAutoFixes"]
+            ),
             text_document_sync: Proto::Interface::TextDocumentSyncOptions.new(
               change: Proto::Constant::TextDocumentSyncKind::FULL,
               open_close: true

--- a/test/standard/runners/lsp_test.rb
+++ b/test/standard/runners/lsp_test.rb
@@ -19,7 +19,7 @@ class Standard::Runners::LspTest < UnitTest
         textDocumentSync: {openClose: true, change: 1},
         documentFormattingProvider: true,
         diagnosticProvider: true,
-        executeCommandProvider: true
+        executeCommandProvider: {commands: ["standardRuby.formatAutoFixes"]}
       }},
       jsonrpc: "2.0"
     }

